apps/wallets/smart-wallet/react/src/app/utils/createAAWallet/createViemWallet.ts

```diff
@@ -25,26 +29,12 @@ export const createViemAAWallet = async (isProd: boolean) => {
               clientApiKey: process.env.REACT_APP_CROSSMINT_API_KEY_STG || "",
           });
 
-    let mnemonic = localStorage.getItem(`mnemonic-${email}`);
-
-    if (!mnemonic) {
-        try {
-            mnemonic = "legal winner thank year wave sausage worth useful legal winner thank yellow";
-            localStorage.setItem(`mnemonic-${email}`, mnemonic);
-        } catch (error) {
-            console.error("Error generating mnemonic:", error);
-            throw error;
-        }
-    }
-
-    const account = mnemonicToAccount(mnemonic) as any;
     // NOTE: Do NOT do this in production. This is just for demo purposes.
     // Proper storage of private key material is critical.
     // Crossmint supports several secure signer options, documented later in the guide.
-
     const signer: ViemAccount = {
         type: "VIEM_ACCOUNT",
-        account,
+        account: privateKeyToAccount(privateKey) as any,
```

Comment:
Maybe worth spending a few minutes to see if there's a more specific casting we can do.

Reply:
I wonder if updating the viem package to the same one the sdk uses would help here?